infrastructure/helm/quantumserverless/charts/gateway/templates/deployment.yaml

```diff
@@ -27,6 +27,10 @@ spec:
       serviceAccountName: {{ include "gateway.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
+        - name: waitpostresql
+          image: actions/pg_isready
+          command: ['sh', '-c', 'until pg_isready -U {{ .Values.database.user | quote }} -d "dbname={{ .Values.database.name }}" -h {{ .Release.Name }}-postgresql -p {{ .Values.database.port | quote }}; do echo waiting for myservice; sleep 2; done']
```

Comment:
Correct me if I'm wrong @akihikokuroda , @psschwei but now from #505 we are not using or requiring the use of {{ .Release.Name }}, right?

Reply:
And thinking in loud. I would suggest to pass the host entirely as we are doing with the user and so on, I don't know what do you think (thinking that we are going to have the DBs outside the cluster, initially).

Reply:
by the way, won't it be required in this case the password?